samples/watson-train-serve/README.md

```diff
@@ -3,56 +3,21 @@
 This pipeline runs training, storing and deploying a Tensorflow model with MNIST handwriting recognition using [IBM Watson Machine Learning](https://www.ibm.com/cloud/machine-learning) service. This example is originated from Kubeflow pipeline's [ibm-samples/watson](https://github.com/kubeflow/pipelines/tree/master/samples/contrib/ibm-samples/watson) example.
 
 ## Prerequisites 
-- Install [Kubeflow 1.0.2+](https://www.kubeflow.org/docs/started/getting-started/) and connect the cluster to the current shell with `kubectl`
-- Install [Tekton 0.11.3](https://github.com/tektoncd/pipeline/releases/tag/v0.11.3) and [Tekton CLI](https://github.com/tektoncd/cli)
-    - For KFP, we shouldn't modify the default work directory for any component. Therefore, please run the below command to disable the [home and work directory overwrite](https://github.com/tektoncd/pipeline/blob/master/docs/install.md#customizing-the-pipelines-controller-behavior) from Tekton default.
-        ```shell
-        kubectl patch cm feature-flags -n tekton-pipelines -p '{"data":{"disable-home-env-overwrite":"true","disable-working-directory-overwrite":"true"}}'
-        ```
-- Install [kfp-tekton](/sdk/README.md#steps) SDK
+- Install [KFP Tekton prerequisites](/samples/README.md)
 
 - Install [Watson Requirements](https://github.com/kubeflow/pipelines/tree/master/samples/contrib/ibm-samples/watson#requirements) 
 
 ## Instructions
 
-1. Compile and apply the Watson ML pipeline
+1. Compile the Watson ML pipeline
 ```shell
-dsl-compile-tekton --py watson_train_serve_pipeline.py --output watson-train-server.yaml
-kubectl apply -f watson-train-server.yaml 
+python watson_train_serve_pipeline.py
```

Comment:
I honestly don't think this is better. This is still using the command line, but hides how the compilation happens inside the Python DSL scripts main method. And it is no longer clear what the output file will be called.